### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by izzbizz**

### CODE REVIEW

File: content/blog/rag-deployment/index.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  
+
+Again, Kubernetes is our friend. It receives the requests through the [Haystack REST API](https://docs.haystack.deepset.ai/docs/rest_api) endpoint and creates replicas of the query pipeline when many requests come in at once. It also ensures that these pipeline replicas have the necessary hardware resources available when needed. The queries are processed by the pipelines and the retrieval-augmented, LLM-generated responses are returned to your application, where your users can interact with them.
+
+![Sketch of multiple replicas of the RAG pipeline, receiving many user queries and communicating with the database.](replicas.png)
+
+While the nitty-gritty technical details of scaling are handled by Kubernetes, we have the ability to tweak it based on the type of pipeline. To do this, it’s useful to think about the following questions:
+
+  
+
```
Comment (TuanaCelik) [inline content/blog/rag-deployment/index.md:126]: And 1 question: What's the role of model deployment services here? Does it make sense to mention them? E.g.: If I go with Sagemaker, will sagemaker handle scaling model requests? That incurs a cost but maybe it's worth it?
@izzbizz @ArzelaAscoIi

File: content/blog/rag-deployment/index.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+While the first two points are handled by an indexing pipeline, the second is handled by a query pipeline. In Haystack, you define both in the same yaml file.
+
+## Moving to Production
+
+There are many tools you can use on your way to production. The exact choice depends on your needs and preferences – and if you have deployed a system before, you probably already have a preferred setup. Either way, let’s take stock of what you need for a successful deployment:
+
+  
+
+-   A production-ready, managed database, such as [OpenSearch](https://opensearch.org/) as a service. The advantage of using a database managed by a third party is that they take care of all the complex stuff like database maintenance and security for you.
```
[the remark this answers is not in the document]
Comment (ArzelaAscoIi) [reply content/blog/rag-deployment/index.md:67]: Yes! totally makes sense!

File: content/blog/rag-deployment/index.md (hunk 1 above)
Comment (ArzelaAscoIi) [reply content/blog/rag-deployment/index.md:126]: Yes. We might want to be a bit careful with not create confusion for the reader, but what do you think about adding a comment about: "Considering an additional model hosting services like sagemaker (or hf  inference) might be helpful to spearately scale model inference"

Comment (ArzelaAscoIi) [reply content/blog/rag-deployment/index.md:126]: not 100% sure about the wording

**Review by ArzelaAscoIi — APPROVED** [review]

**Review by TuanaCelik — APPROVED** [review]

**Pull request merged**